script.js

```diff
@@ -1,4 +1,106 @@
+var stage = 'chooseNoOfPlayer'
+var noOfPlayers = 0
+var player = 1
+var noOfDice = 0
+var numTotal = []
+var scoreTotal = []
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+console.log(`player ${player}`)
+var myOutputValue = ''
+  if (stage == 'chooseNoOfPlayer') {
+    if (Number.isNaN(Number(input)) || Number(input) < 2 || Number(input)%1 != 0) {
+      myOutputValue = `Sorry, please enter a whole number greater than 1.`
+    } else {
+      noOfPlayers = Number(input)
+      for (let i=0; i<noOfPlayers; i++) {
+        scoreTotal.push([0])
+      }
+      myOutputValue = `${noOfPlayers} players it is! Now pick the number of dice you wish to play with this round.`
+      if (player == 1) {
+        stage = 'chooseNoOfDice'
+      } else {
+        stage == 'roll'
+      }
+    }
+  }
+
+  else if (stage == 'chooseNoOfDice') {
+    if (Number.isNaN(Number(input)) || Number(input) < 1 || Number(input)%1 != 0) {
```

Comment:
You could put these into a function that returns true or false so something like:
`if (inputIsValid(input)) { `

where `inputIsValid` is a function you create that returns a true or false value